apps/storybook.namekit.io/.storybook/preview.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import "@namehash/nameguard-react/styles.css";
+import "@namehash/namekit-react/styles.css";
+
+import { Preview } from "@storybook/react";
+
+import {
+  Title,
+  Subtitle,
+  Description,
+  // Primary, // TODO: Hiding `Primary` for now until we can fix the issue with `Controls`.
+  // Controls, // TODO: See comment below about why we're hiding this for now.
+  Stories,
+} from "@storybook/blocks";
+
+import "../global.css";
+
+const preview: Preview = {
+  parameters: {
+    options: {
+      storySort: {
+        order: ["Nameguard", "Namekit", "ENS Utils", "ENS Webfont"],
+      },
+    },
+    docs: {
```

Comment:
Needed to define a custom `docs` param here to hide the `<Primary>` and `<Controls>` sections that are rendered on "Docs" pages by default.

Please see comments below for details.

I already spent several hours on this. Let's please not get stuck on this any more for now and focus on advancing this PR so that we can close it out as soon as possible.